app/controllers/payments_controller.rb

```diff
@@ -27,5 +27,4 @@ def create
       @invoice.update stripe_charge_id: charge.id
       redirect_to @invoice, notice: 'Thanks for the money lol'
   end
-
```

Comment:
@nTamura look at the above. Need to fix the indenting and syntax. You are using hash rockets and colons. Pick one and stick to it. Preferrably the colons, thats the new accepted style...

For example
```
     customer = Stripe::Customer.create(
       description: "Customer for #{current_user.id} | #{current_user.email}",
       source: params[:stripe_token]
     )
```